src/json.cpp

```diff
@@ -402,7 +402,7 @@ char *json_encode_string(const char *str)
   try {
     emit_string(&sb, str);
   }
-  catch (std::exception) {
+  catch (std::exception&) {
```

Comment:
Might be good to PR this upstream (if you didn't already)

Reply:
Do you know where the upstream repo is?

Reply:
@joeyadams did you have this code hosted somewhere for this patch to be submitted?